README.md

```diff
@@ -200,16 +200,31 @@ These options may also be configured from knife.rb, as in this example:
     :kerberos_service             The Kerberos service used for authentication
     :ca_trust_file                The Certificate Authority (CA) trust file used for SSL transport
 
+#### Options to configure WinRM for Bootstrapping a Windows Node
+
+    :winrm_max_timeout           Set winrm max timeout in miliseconds
```

Comment:
Spelling of 'milliseconds', also see other comment that we should use minutes instead of milliseconds.